test/battle/move_effect/acrobatics.c

```diff
@@ -1,5 +1,45 @@
 #include "global.h"
 #include "test/battle.h"
 
-TO_DO_BATTLE_TEST("Acrobatics doubles in power if the user has no held item");
-TO_DO_BATTLE_TEST("Acrobatics still doubles in power when Flying Gem is consumed");
+ASSUMPTIONS
+{
+    ASSUME(gMovesInfo[MOVE_ACROBATICS].effect == EFFECT_ACROBATICS);
+    ASSUME(gMovesInfo[MOVE_ACROBATICS].type == TYPE_FLYING);
+}
+
+SINGLE_BATTLE_TEST("Acrobatics doubles in power if the user has no held item", s16 damage)
+{
+    u16 heldItem;
+    PARAMETRIZE{ heldItem = ITEM_POTION; }
```

Comment:
Global remark, but we always separate ``PARAMETRIZE`` and the following brace with a space